js/coinone.js

```diff
@@ -209,16 +209,16 @@ module.exports = class coinone extends Exchange {
 
     parseTicker (ticker, market = undefined) {
         const timestamp = this.safeTimestamp (ticker, 'timestamp');
-        const open = this.safeNumber (ticker, 'first');
-        const last = this.safeNumber (ticker, 'last');
-        const previousClose = this.safeNumber (ticker, 'yesterday_last');
+        const open = this.safeString (ticker, 'first');
```

Comment:
If possible, please always add a response sample to `fetchTicker()` / `parseTicker()` if it's missing. See how it is formatted in the other exchanges. In `fetchTicker()` we comment the entire response, and in `parseTicker()` we comment just one ticker structure that is being parsed.